packages/tables/resources/views/index.blade.php

```diff
@@ -222,7 +222,7 @@ class="flex items-center justify-between p-2 h-14"
                 @if ($isGlobalSearchVisible || $hasFiltersPopover || $isColumnToggleFormVisible)
                     <div class="flex items-center justify-end w-full gap-2 md:max-w-md">
                         @if ($isGlobalSearchVisible)
-                            <div class="flex items-center justify-end flex-1">
+                            <div class="filament-tables-search-container flex items-center justify-end flex-1">
```

Comment:
Maybe this is better?

```suggestion
                            <div class="filament-tables-header-search-container flex items-center justify-end flex-1">
```

Reply:
We already have `filament-tables-search-input` so i think this makes sense without `-header`

Reply:
For v3 I'm looking to merge the different search inputs we have. Therefore it would make sense to prefix the container with `-header` as `filament-tables-search-input` will potentially be renamed to e.g. `filament-search-input`.